bot/resources/tags/cooldown.md

```diff
@@ -0,0 +1,22 @@
+**Cooldowns**
+
+Cooldowns are used in discord.py to rate-limit.
+
+```python
+from discord.ext import commands
+
+class SomeCog(commands.Cog):
+    def __init__(self):
+        self._cd = commands.CooldownMapping.from_cooldown(1.0, 60.0, commands.BucketType.user)
+
+    async def cog_check(self, ctx):
```

Comment:
I'm not incredibly well versed, but why this approach instead of the decorator based approach? For beginners that may come across easier.

Reply:
This isn't for specifically ratelimiting commands. It's to manage your own custom cooldowns, whether it be emojis in chat, links, or anything else.

Reply:
Ah gotcha